coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorConf.java

```diff
@@ -92,7 +93,8 @@ public class CoordinatorConf extends RssBaseConf {
           .asList()
           .defaultValues(
               "org.apache.uniffle.coordinator.access.checker.AccessClusterLoadChecker",
-              "org.apache.uniffle.coordinator.access.checker.AccessQuotaChecker")
+              "org.apache.uniffle.coordinator.access.checker.AccessQuotaChecker",
```

Comment:
Could you use similar way to refactor the code like ` AccessSupportRssChecker.class.getCanonicalName()`?